bin/gwc/gwcomp.ml

```diff
@@ -1171,7 +1171,7 @@ let read_family ic fname = function
               divorce;
               fevents = [];
               comment = comm;
-              origin_file = fname;
+              origin_file = Filename.basename fname;
```

Comment:
why do we need `Filename.basename` here?

Does it still work if `fname` is a file path not in the current directory?

Reply:
The only important information is the filename.gw!
At output time, the full path is driven by `-o` or `-odir` (see below)